src/components/cards/CreatePool/PoolFees.vue

```diff
@@ -82,12 +82,15 @@ const feeOptions = FIXED_FEE_OPTIONS.map(option => {
  * FUNCTIONS
  */
 function onFixedInput(val: string): void {
+  fee.value = '';
   initialFee.value = '0';
   initialFee.value = val;
```

Comment:
hm, don't understand here why initialFee is set to 0 first, and then to value

Reply:
I didn't want to change `initialFee.value = '0';` because it was already like that and didn't want to change the logic, although I agree it doesn't make much sense since it's set to `val` straight after, I'll remove. Setting `fee.value` to nothing here is because `fee.value` actually represents the custom input value, it's just badly named.